hatasmota/camera.py

```diff
@@ -0,0 +1,131 @@
+"""Tasmota camera."""
+
+from __future__ import annotations
+
+from collections.abc import Awaitable
+from dataclasses import dataclass
+import logging
+from typing import Any
+
+from aiohttp import ClientResponse, ClientSession
+
+from .const import CONF_DEEP_SLEEP, CONF_IP, CONF_MAC
+from .entity import (
+    TasmotaAvailability,
+    TasmotaAvailabilityConfig,
+    TasmotaEntity,
+    TasmotaEntityConfig,
+)
+from .mqtt import ReceiveMessage
+from .utils import (
+    config_get_state_offline,
+    config_get_state_online,
+    get_topic_command,
+    get_topic_command_state,
+    get_topic_stat_result,
+    get_topic_tele_sensor,
+    get_topic_tele_state,
+    get_topic_tele_will,
+    get_value_by_path,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass(frozen=True, kw_only=True)
+class TasmotaCameraConfig(TasmotaAvailabilityConfig, TasmotaEntityConfig):
+    """Tasmota camera configuation."""
+
+    idx: int
```

Comment:
Does tasmota support more than a single camera? If not, can we skip this, it seems to be hardcode to 0 below?

Reply:
Removed.